src/main/java/uk/ac/cam/cl/dtg/segue/api/EmailFacade.java

```diff
@@ -488,10 +489,25 @@ public final Response sendEmailsToUserIds(@Context final HttpServletRequest requ
 
         try {
             RegisteredUserDTO sender = this.userManager.getCurrentRegisteredUser(request);
-            if (!isUserAnAdmin(userManager, sender)) {
+            if (!isUserAnAdminOrEventManager(userManager, sender)) {
                 return SegueErrorResponse.getIncorrectRoleResponse();
             }
 
+            if (isUserAnEventManager(userManager, sender)) {
+                if (!emailType.equals(EmailType.EVENTS)) {
```

Comment:
Where did this restriction come from? I think it's ok for now, but I foresee it causing issues.